docs/tutorial/index.rst

```diff
@@ -282,6 +285,270 @@ you will access the build logs,
+- A new ``stable`` version, pointing to the ``origin/1.0.x`` branch, and also "Public".
+
+.. figure:: /_static/images/tutorial/active-versions.png
+   :width: 80%
+   :align: center
+   :alt: List of active versions of the project
+
+   List of active versions of the project
+
+Right after you created your branch,
+Read the Docs created a new special version called ``stable`` pointing to it,
+and started building it. When the build finishes,
+the ``stable`` version will be listed in the :term:`flyout menu`
+and your readers will be able to choose it.
+
+Now you might want to set ``stable`` as the *default version*,
+rather than ``latest``.
+This has the advantage that users will see the documentation
+corresponding to the most recent release of your software,
+while they can optionally switch to the development version.
+
+For that, go to the "Advanced Settings" link under the "⚙ Admin" menu of your project home,
+choose ``stable`` in the "Default version*" dropdown,
+and hit :guilabel:`Save` at the bottom.
+From now on, every time a user visits the :term:`root URL` of your documentation,
+they will get redirected to the ``stable`` version of it.
+
+Modifying versions
+~~~~~~~~~~~~~~~~~~
+
+Both ``latest`` and ``stable`` are now *active*, which means that
+they are visible for users, and new builds can be triggered for them.
+In addition to these, Read the Docs also created an *inactive* ``1.0.x``
+version, which will always point to the ``1.0.x`` branch of your repository.
+
+.. figure:: /_static/images/tutorial/inactive-versions.png
+   :width: 80%
+   :align: center
+   :alt: List of inactive versions of the project
+
+   List of inactive versions of the project
+
+Let's activate the ``1.0.x`` version.
+For that, go to the "Versions" on your :term:`project home`,
+locate ``1.0.x`` under "Activate a version",
+and click on the :guilabel:`Activate` button.
+This will take you to a new page with two checkboxes,
+"Active" and "Hidden". Check only "Active",
+and click :guilabel:`Save`.
+
+After you do this, ``1.0.x`` will appear on the "Active Versions" section,
+and a new build will be triggered for it.
+
+Show a warning for old versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When your project matures, the number of versions might increase.
+Sometimes you will want to warn your readers
+when they are browsing an old or outdated version of your documentation.
+
+To showcase how to do that, let's create a ``2.0`` version of the code:
+navigate to your GitHub repository, click on the branch selector,
+type :kbd:`2.0.x`, and click on "Create branch: 2.0.x from 'main'".
+This will trigger two things:
+
+- Since ``2.0.x`` is your newest branch, ``stable`` will switch to tracking it.
+- A new ``2.0.x`` version will be created on your Read the Docs project.
+
+.. warning::
+
+   At the time of writing, this ``2.0.x`` version will become active automatically.
+   This, however, is unintended behaviour under investigation.
```

Comment:
This is done if your project follows semver, each new stable version is activated and built automatically. If you create a new 1.5, it won't activate, but if you create a 2.1, it will.

Reply:
This is the "issue" I mentioned yesterday. Isn't it weird that I have to activate the first one, but the second one gets activated automatically? And also, not all "second versions" are activated always, from my experiments yesterday. What part of the code determines that?

Reply:
there is some mention of this in the note at the end of this section https://docs.readthedocs.io/en/stable/automation-rules.html#actions

The code is at (inner if) https://github.com/readthedocs/readthedocs.org/blob/b68a92106b5926c7e708f77aa3d04e2f30b0a24b/readthedocs/builds/tasks.py#L324-L345

We only activate that version if the previous one was active (1.0 in your example was activate, so 2.0 will be marked as active as well).

Reply:
Ah, I see:

https://github.com/readthedocs/readthedocs.org/blob/b68a92106b5926c7e708f77aa3d04e2f30b0a24b/readthedocs/builds/tasks.py#L272-L278

Will update the tutorial with this information.

Reply:
I'm also fine deleting this code. I think it's confusing, and users can now create this effect with automation actions. I'd like to remove as much of this magic as possible, since it's confusing.